### PR TITLE
fix: Menu language

### PR DESCRIPTION
**Pull request by ChefMomota**

### CODE REVIEW

File: src/components/Menu/hooks/useMenuItems.ts [hunk 1]
```diff
@@ -11,9 +11,7 @@ export const useMenuItems = (): ConfigMenuItemsType[] => {
   } = useTranslation()
   const menuItemsStatus = useMenuItemsStatus()
 
-  const menuItems = useMemo(() => {
-    return config(t, languageCode)
-  }, [t, languageCode])
+  const menuItems = config(t, languageCode)
```
Comment (chef-jojo) [inline src/components/Menu/hooks/useMenuItems.ts:14]: can you try to move this line under the useMemo below?

Comment (ChefMomota) [reply src/components/Menu/hooks/useMenuItems.ts]: Will have the same issue with the current one.

Comment (chef-jojo) [reply src/components/Menu/hooks/useMenuItems.ts:14]: so weird... the menuItems will not be memo at current, i'll handle it later

**Review by chef-jojo — APPROVED** [review]

File: src/components/Menu/hooks/useMenuItems.ts (hunk 1 above)
Comment (memoyil) [reply src/components/Menu/hooks/useMenuItems.ts:14]: What was the issue ?

Comment (chef-jojo) [reply src/components/Menu/hooks/useMenuItems.ts:14]: @memoyil 
first load with other lang doesn't work on menu items. maybe we need to use lodash memoize
yet it's weird that the useMemo doesn't trigger when locale changed

Comment (memoyil) [reply src/components/Menu/hooks/useMenuItems.ts:14]: I think the root cause of the issue is related to translation function. Can you review this ? https://github.com/pancakeswap/pancake-frontend/pull/4135

**Pull request merged**